### PR TITLE
[color-picker] fix: mobile style fix

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: e2e/tests/text.e2e.js [hunk 1]
```diff
@@ -313,6 +318,20 @@ describe('text', () => {
   });
 });
 
+describe('text color mobile', () => {
+  before(function() {
+    cy.eyesOpen({
+      appName: 'Text',
+      testName: this.test.parent.title,
+      browser: DEFAULT_DESKTOP_MOBILE,
```
Comment (tamrat-bay) [inline e2e/tests/text.e2e.js:326]: Change to DEFAULT_MOBILE_BROWSERS

File: e2e/tests/text.e2e.js [hunk 2]
```diff
@@ -313,6 +318,20 @@ describe('text', () => {
...
+    });
+  });
+  it('allow to change text color on mobile', function() {
+    cy.switchToMobile();
```
Comment (tamrat-bay) [inline e2e/tests/text.e2e.js:330]: move cy.switchToMobile() before th "it"

File: packages/plugin-commons/web/src/Components/ColorPicker/ColorPicker.jsx [hunk 3]
```diff
@@ -75,7 +75,13 @@ class ColorPicker extends PureComponent {
 
   renderColorButtons(colors, attributes) {
     const { styles } = this;
-    const { schemeColor } = this.props;
+    const { schemeColor, isMobile } = this.props;
+    const userColors = !attributes;
+
+    if (isMobile && colors.length === 6 && !userColors) {
+      colors.pop();
```
Comment (shaulgo) [inline packages/plugin-commons/web/src/Components/ColorPicker/ColorPicker.jsx:82]: what does this do? Doesn't look like something that should happen in the render function

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: e2e/tests/text.e2e.js [hunk 4]
```diff
@@ -1,9 +1,24 @@
...
 /*global cy*/
 import { INLINE_TOOLBAR_BUTTONS } from '../cypress/dataHooks';
-import { DEFAULT_DESKTOP_BROWSERS, FIREFOX_BROWSER } from './settings';
+import { DEFAULT_DESKTOP_BROWSERS, FIREFOX_BROWSER, MOBILE_BROWSERS } from './settings';
 import { usePlugins, usePluginsConfig, plugins } from '../cypress/testAppConfig';
 
+const changeTextColor = title => {
+  cy.loadRicosEditorAndViewer('plain')
+    .setTextStyle(INLINE_TOOLBAR_BUTTONS.COLOR, [20, 15])
+    .addColor();
```
Comment (maimonav) [inline e2e/tests/text.e2e.js:11]: Rename `add color` to `openCustomColorModal`.

File: packages/plugin-commons/web/src/Components/ColorPicker/ColorPicker.jsx [hunk 5]
```diff
@@ -75,7 +75,13 @@ class ColorPicker extends PureComponent {
 
   renderColorButtons(colors, attributes) {
     const { styles } = this;
-    const { schemeColor } = this.props;
+    const { schemeColor, isMobile } = this.props;
+    const userColors = !attributes;
+
```
Comment (maimonav) [inline packages/plugin-commons/web/src/Components/ColorPicker/ColorPicker.jsx:80]: Move color's logic to parent component.
Parent in `isMobile` mode will pass array of 5 colors, and 6 colors in desktop.
Make sure to clean all other logic here that related.

File: packages/plugin-text-color/web/src/toolbar/TextColorPanel.jsx [hunk 6]
```diff
@@ -88,17 +96,30 @@ export default class TextColorPanel extends Component {
...
           mergedStyles,
         }) => (
-          <div className={mergedStyles.colorPicker_palette}>
-            <div className={mergedStyles.colorPicker_buttons_container}>
-              {renderPalette()}
-              {renderUserColors()}
+          <>
+            {isMobile && (
+              <>
+                <div className={mergedStyles.colorPicker_header}>{t(colorPickerHeader)}</div>
```
Comment (maimonav) [inline packages/plugin-text-color/web/src/toolbar/TextColorPanel.jsx:102]: Call `t` function in parent and pass here translated header text.

**Review by maimonav — CHANGES_REQUESTED** [review]
1. Size of Hex input text.
2. Check Surge, multiple css issues - (reset to default & new color), colors not centered in mobile.
3. Comments from PR.

**Review by maimonav — APPROVED** [review]

**Pull request merged**